src/algebra/star/basic.lean

```diff
@@ -211,29 +211,44 @@ instance star_ring.to_star_add_monoid [semiring R] [star_ring R] : star_add_mono
 def star_ring_equiv [semiring R] [star_ring R] : R ≃+* Rᵐᵒᵖ :=
 { to_fun := λ x, mul_opposite.op (star x),
   ..star_add_equiv.trans (mul_opposite.op_add_equiv : R ≃+ Rᵐᵒᵖ),
-  ..star_mul_equiv}
+  ..star_mul_equiv }
 
-/-- `star` as a `ring_aut` for commutative `R`. This is used to denote complex
-conjugation, and is available under the notation `conj` in the locale `complex_conjugate` -/
+/-- `star` as a ring automorphism, for commutative `R`. -/
 def star_ring_aut [comm_semiring R] [star_ring R] : ring_aut R :=
```

Comment:
```suggestion
@[simps apply]
def star_ring_aut [comm_semiring R] [star_ring R] : ring_aut R :=
```
We removed this originally to make `conj` happy, but that's no longer relevant, so it should be safe to add it back.